tracer/src/Datadog.Trace/OpenTelemetry/Logs/OtlpExporter.cs

```diff
@@ -0,0 +1,261 @@
+        public bool Shutdown(int timeoutMilliseconds)
+        {
+            try
+            {
+                _httpClient.CancelPendingRequests();
+                _httpClient.Dispose();
+                Log.Debug("OTLP Log Exporter shutdown completed.");
+                return true;
+            }
+            catch (Exception ex)
+            {
+                Log.Error(ex, "Error during OTLP Log Exporter shutdown.");
+                return false;
+            }
+        }
+
+        private static HttpClient CreateHttpClient()
+        {
+#if NET6_0_OR_GREATER
+            var tcpHandler = new SocketsHttpHandler
+            {
+                AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate
+            };
+
+            return new HttpClient(tcpHandler)
+            {
+                DefaultRequestVersion = HttpVersion.Version20,
+                DefaultVersionPolicy = HttpVersionPolicy.RequestVersionOrHigher
+            };
+#else
+            return new HttpClient
+            {
+                DefaultRequestVersion = HttpVersion.Version20,
+            };
+#endif
+        }
+
+        private async Task<bool> SendOtlpRequest(IReadOnlyList<LogPoint> logs)
+        {
+            try
+            {
+                // Serialize logs to protobuf format using vendored OpenTelemetry protobuf utilities
+                // For gRPC, reserve 5 bytes at the start for the frame header (added later)
+                // For HTTP, start at position 0
+                var startPosition = _protocol == OtlpProtocol.Grpc ? 5 : 0;
+                var otlpPayload = OtlpLogsSerializer.SerializeLogs(logs, _settings, startPosition);
+
+                return _protocol switch
+                {
+                    OtlpProtocol.HttpProtobuf => await SendHttpProtobufRequest(otlpPayload).ConfigureAwait(false),
+                    OtlpProtocol.HttpJson => await SendHttpJsonRequest(otlpPayload).ConfigureAwait(false),
+                    OtlpProtocol.Grpc => await SendGrpcRequest(otlpPayload).ConfigureAwait(false),
+                    _ => await SendHttpProtobufRequest(otlpPayload).ConfigureAwait(false)
+                };
+            }
+            catch (Exception ex)
+            {
+                Log.Error(ex, "Error sending OTLP request.");
+                return false;
+            }
+        }
+
+        private async Task<bool> SendHttpJsonRequest(byte[] otlpPayload)
+        {
+            Log.Warning("HTTP/JSON protocol is not yet implemented for logs, falling back to HTTP/protobuf");
+            return await SendHttpProtobufRequest(otlpPayload).ConfigureAwait(false);
+        }
+
+        private async Task<bool> SendHttpProtobufRequest(byte[] otlpPayload)
+        {
+            if (_httpExportClient is null)
+            {
+                Log.Warning("HTTP/Protobuf selected but HTTP client is not initialized; cannot send logs.");
+                return false;
+            }
```

Comment:
This is another smell IMO. We should refactor this type so we _can't_ be in this "invalid" state